bcda/cclf/cclf.go

```diff
@@ -469,17 +473,42 @@ func sortCCLFFiles(cclfmap *map[string][]*cclfFileMetadata, skipped *int) filepa
 			return nil
 		}
 
-		// if we get multiple sets of files relating to the same aco for attribution purposes, concat the year
-		key := fmt.Sprintf(metadata.acoID+"_%d", metadata.perfYear)
-		if (*cclfmap)[key] != nil {
-			(*cclfmap)[key] = append((*cclfmap)[key], &metadata)
+		if (*cclfMap)[metadata.acoID] != nil {
+			if (*cclfMap)[metadata.acoID][metadata.perfYear] != nil {
+				(*cclfMap)[metadata.acoID][metadata.perfYear] = append((*cclfMap)[metadata.acoID][metadata.perfYear], &metadata)
+			} else {
+				(*cclfMap)[metadata.acoID][metadata.perfYear] = []*cclfFileMetadata{&metadata}
+			}
 		} else {
-			(*cclfmap)[key] = []*cclfFileMetadata{&metadata}
+			(*cclfMap)[metadata.acoID] = map[int][]*cclfFileMetadata{metadata.perfYear: []*cclfFileMetadata{&metadata}}
 		}
+
 		return nil
 	}
 }
 
+func orderACOs(cclfMap *map[string]map[int][]*cclfFileMetadata) []string {
+	var acoOrder []string
+
+	if priorityACOList := os.Getenv("CCLF_PRIORITY_ACO_CMS_IDS"); priorityACOList != "" {
```

Comment:
Do we need an accompanying ops ticket to get this set in `prod`?

Reply:
Indeed we do! https://jira.cms.gov/browse/BCDA-1951